main/guides/getting-started/README.md

```diff
@@ -43,12 +44,20 @@ The Agoric SDK is supported on <a href="https://en.wikipedia.org/wiki/Linux">Lin
 - To launch a terminal on Linux, use the **Terminal** application.
 - To access WSL from Windows, visit the [WSL documentation](https://docs.microsoft.com/en-us/windows/wsl/).
 
-## Install Node.js 16.19.1 or Higher
+## Install jQuery
```

Comment:
```suggestion
## Install jq
```

`jq` is just [jq](https://stedolan.github.io/jq/). jQuery is https://jquery.com/

Reply:
Fixed. (And moved to another PR)